File: 02 Binary/Number of 1 Bits/Count1bits.go
package main

import "fmt"

func hammingWeight(n int) int {
	b := fmt.Sprintf("%b", n)
	var count int
	for _, v := range b {
		if v == '1' {
			count++
		}
	}
	return count
}

func main() {
	var t int
	fmt.Scan(&t)
	for t > 0 {
		t--
		var n int
		fmt.Scan(&n)
		fmt.Println(hammingWeight(n))
	}
}

File: 02 Binary/Number of 1 Bits/Count1bits.java
import java.util.Scanner;

class Solution {
    public int hammingWeight(int n) {
        int count = 0;
        while (n != 0) {
            count += n & 1;
            n >>= 1;
        }
        return count;
    }
}

public class Count1bits {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while (t > 0) {
            int n = sc.nextInt();
            Solution obj = new Solution();
            System.out.println(obj.hammingWeight(n));
            t--;
        }
        sc.close();
    }
}